sw/device/tests/BUILD

```diff
@@ -2452,7 +2451,6 @@ opentitan_functest(
         # This test will need hyperdebug once we start using multi-lane
         # SPI modes.
         #interface = "hyper310",
-        #tags = ["manual"],
```

Comment:
Are you completely removing this tag? If so, we should also remove the other references, such as in the CI yml files.